integrationtest/docker/kafka/LICENSE

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
We don't keep a separate license file for sub-folders. Instead, we add license headers to source files (except README).